### DISCUSSION TITLE
Task Lists: SHIFT-TAB to un-indent

### DISCUSSION THREAD

**Discussion by bseib** [community/community#5623] [Projects and Issues]
When I have a list that looks like:

 - [ ] foo
 - [ ] bar
 - [ ] baz
   - quux

When I edit this with the caret at the end of "quux"

![image](https://user-images.githubusercontent.com/1313561/133121861-dd887c4b-2493-4773-845e-117f1b6f4475.png)

pressing a carriage return yields:

![image](https://user-images.githubusercontent.com/1313561/133121818-8d2240c8-d52e-42a1-9f27-4691e38cd663.png)

where the bullet `-` is indented the same as the previous line.

At this moment my instinct is to type <kbd>TAB</kbd> or <kbd>SHIFT</kbd>+<kbd>TAB</kbd> to either indent or un-indent this bullet by one level.

Yes, I know <kbd>TAB</kbd> already has expected behavior in browsers, and potentially accessibility issues if that behavior is changed. And yes this is only milliseconds of editing time saved. But I thought I'd still ask and see what might be possible. Perhaps there's another way to provide this behavior in the editor no matter who the person is at the keyboard.

**Comment by gberche-orange** [comment]
Workaround might be to edit the markdown in a IDE (with proper UX for indentation) and then (automatically) import the content into github UI
See related https://youtrack.jetbrains.com/issue/IDEA-342299/add-support-to-create-edit-comment-github-issues
